projects/ui/src/shim.cds-core.scss

```diff
@@ -472,17 +472,34 @@
 
   // Checkbox
-  --clr-forms-checkbox-label-color: var(--clr-forms-label-color); // indirectly mapped to CDS token
-  --clr-forms-checkbox-background-color: var(--cds-alias-status-info); // var(--clr-color-action-600);
-  --clr-forms-checkbox-indeterminate-border-color: var(--cds-alias-status-info); // var(--clr-color-action-600);
-  --clr-forms-checkbox-mark-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100)); // var(--clr-color-neutral-0);
-  --clr-forms-checkbox-disabled-background-color: var(--clr-form-disabled-background-color);
-  --clr-forms-checkbox-disabled-mark-color: var(--cds-alias-typography-color-100, var(--cds-global-typography-color-100)); // var(--clr-color-neutral-1000);
+  --clr-forms-checkbox-label-color: var(--clr-forms-label-color); // indirectly mapped to CDS token --cds-global-typography-color-400
+  --clr-forms-checkbox-background-color: var(--cds-alias-object-interaction-background-highlight); // var(--clr-color-action-600);
+  --clr-forms-checkbox-indeterminate-border-color: var(--cds-alias-object-interaction-background-highlight); // var(--clr-color-action-600);
+  --clr-forms-checkbox-mark-color: var(--cds-alias-object-interaction-background); // var(--clr-color-neutral-0);
+
+  --clr-forms-checkbox-disabled-label-color: var(--clr-forms-label-disabled-color); // indirectly mapped to CDS token --cds-global-typography-color-400
+  --clr-forms-checkbox-disabled-background-color: var(--clr-btn-default-bg-color); // indirectly mapped to CDS token --cds-alias-object-interaction-background
+  --clr-forms-checkbox-disabled-checked-background-color: var(--cds-alias-status-disabled); // indirectly mapped to CDS token --cds-alias-object-interaction-background
+  --clr-forms-checkbox-disabled-border-color: var(--cds-alias-status-disabled); // var(--clr-color-action-600);
+  --clr-forms-checkbox-disabled-indeterminate-color: var(--cds-alias-status-disabled); // var(--clr-color-action-600);
+  --clr-forms-checkbox-disabled-mark-color: var(--cds-alias-object-interaction-background); // var(--clr-color-neutral-1000);
+
+  --clr-forms-checkbox-line-height: var(--cds-global-space-9);
+  --clr-forms-checkbox-padding-left: var(--cds-global-space-9);
+  --clr-forms-checkbox-label-height: var(--cds-global-space-9);
 
   // Radio
-  --clr-forms-radio-label-color: var(--clr-forms-checkbox-label-color); // indirectly mapped to CDS token
-  --clr-forms-radio-disabled-background-color: var(--clr-form-disabled-background-color); // indirectly mapped to CDS token
-  --clr-forms-radio-disabled-mark-color: var(--clr-forms-checkbox-disabled-mark-color); // indirectly mapped to CDS token
+  --clr-forms-radio-label-color: var(--clr-forms-label-color); // indirectly mapped to CDS token --cds-global-typography-color-400
+  --clr-forms-radio-label-disabled-color: var(--cds-alias-typography-disabled, var(--cds-global-typography-disabled));
```

Comment:
Should be available trough the added from Will file `token_shim.cds-core.scss` so it no longer needs that fallback.

Reply:
corrected in commit: 628b080